### PR TITLE
Fix invitation landing page

### PR DESCRIPTION
**Pull request by HSalmon**

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -11,6 +11,12 @@ class Users::SessionsController < Devise::SessionsController
   before_filter :load_invitation_from_session, only: :new
   after_filter :set_time_zone_from_javascript, only: :create
 
+  def new
+    super do |user|
```
Comment (gdpelican) [inline app/controllers/users/sessions_controller.rb]: Could be
```
  super { |user| user.email = params[:email] }
```

**Comment by gdpelican** [comment]
Looks good to me!

**Comment by HSalmon** [comment]
@gdpelican, would you mind taking a quick look at the new stuff?

**Comment by gdpelican** [comment]
Looks great!

**Pull request merged**